### ISSUE TITLE
On iPhone 2G, what should be the SRAM is marked as AMC0

### ISSUE THREAD

**Issue by MoochMcGee** [iDroid-Project/openiBoot#39]
I know this because the bootrom writes to this area CONSTANTLY, mostly using it as stack space. A QEMU fork that emulates the iPhone 2G also has SRAM here (https://github.com/winocm/QEMU-s5l89xx-port/blob/master/hw/iphone2g.c#L591) as does my emulator, PurpleSapphire (https://github.com/PurpleEmu/PurpleSapphire)

**Comment by ricky26** [comment]
Hi, I lament to say that this project has been dormant for quite some time so it's unlikely anything's going to get changed.

It's certainly interesting that the define even exists since I can't even find any references other than mapping for it.

**Comment by MoochMcGee** [comment]
Yeah... I know. I just figured I'd document it here for posterity.

**Comment by fxsheep** [comment]
seems also the case in A4 
https://github.com/iDroid-Project/openiBoot/blob/e005a4b3936ea09f3f528eb40626db7e8b44632e/plat-a4/includes/hardware/a4.h#L34
